docs/antrea-gke.md

```diff
@@ -0,0 +1,128 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in noEncap mode, in which no encapsulation is required for any kind of traffic (Intra node,
+Inter node etc) and enforcing network policies using OVS.
+
+For details on Antrea noEncap Mode, refer to this. [TODO]
```

Comment:
better to remove this sentence and add it when we have the doc for no encap

Reply:
Done